content/en/docs/installation/supported-releases.md

```diff
@@ -230,18 +230,43 @@ Our testing coverage is:
 The oldest Kubernetes release supported by cert-manager is 1.16, as we want
 to be supporting most commercial Kubernetes offerings.
 
-|   Vendor   | Oldest Kubernetes Release\* |                Other Old Kubernetes Releases                 |
-|:----------:|:---------------------------:|--------------------------------------------------------------|
+|   Vendor   | Oldest Kubernetes Release\* | Other Old Kubernetes Releases                                |
+| :--------: | :-------------------------: | ------------------------------------------------------------ |
 | [EKS][eks] |     1.16 (EOL Jul 2021)     | 1.17 (EOL Sep 2021), 1.18 (EOL Nov 2021), 1.9 (EOF Apr 2022) |
 | [GKE][gke] |     1.17 (EOL Nov 2021)     | 1.18 (EOL Dec 2021), 1.19 (EOL Feb 2022)                     |
 | [AKS][aks] |     1.18 (EOL Jul 2021)     | 1.19 (EOL Aug 2021)                                          |
 
-\*As of June 15, 2021.
+\*As of June 22, 2021.
 
 [eks]: https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar
 [gke]: https://cloud.google.com/kubernetes-engine/docs/release-schedule
 [aks]: https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions#aks-kubernetes-release-calendar
 
+## OpenShift supported versions {#openshift-supported-versions}
+
+With regards to OpenShift Container Platform 3, cert-manager 1.2 is the last
+release to support OpenShift 3.11 (Kubernetes 1.11). Although OpenShift 3.11 is
+still supported by Red Hat until June 2022, keeping support for very old
+versions of Kubernetes had become too much of a burden.
+
+We maintain the following table to remember the mapping between
+OpenShift and Kubernetes versions:
+
+| Version | Kubernetes | EOL            |
+| ------- | ---------- | -------------- |
+| 4.8     | 1.21       | 01 Nov 2022\*  |
+| 4.7     | 1.20       | 01 Jun 2022\*  |
+| 4.6 EUS | 1.19       | 24 May 2022    |
+| 4.6     | 1.19       | 01 Dec 2021\*  |
+| 4.5     | 1.18       | 01 July 2021\* |
+| 4.4     | 1.17       | 24 Feb 2021    |
+| 4.3     | 1.16       | 27 Oct 2020    |
+| 4.2     | 1.14       | 13 Jul 2020    |
+| 4.1     | 1.13       | 05 May 2020    |
```

Comment:
suggestion: I think this table is the most important information in the section - would it makes sense if we move it to the top, above the information about OpenShift 3?

Reply:
Done in 0d3d370.